CRUDRecipeEF.BL.DL/Services/IngredientService.cs

```diff
@@ -22,7 +22,12 @@ public IngredientService(RecipeContext context, IMapper mapper)
         private async Task<Ingredient> GetIngredientByNameIfExists(string name)
         {
             var ingredient = await _context.Ingredients.FirstOrDefaultAsync(i => i.Name.ToLower() == name.ToLower());
-            return ingredient ?? throw new KeyNotFoundException("Ingredient doesnt exist");
+            if (ingredient == null)
+            {
+                throw new KeyNotFoundException("Ingredient doesnt exist");
```

Comment:
How come you changed this?

Reply:
Way easier to see what is going on when running the code in the debugger, that's all :D